src/System.Private.CoreLib/shared/System/Reflection/Emit/OpCodes.cs

```diff
@@ -251,17 +236,20 @@ internal enum OpCodeValues
         Sizeof = 0xfe1c,
         Refanytype = 0xfe1d,
         Readonly_ = 0xfe1e,
-    }
+        // If you add more opcodes here, modify OpCode.Name to handle them correctly
+    };
 
-    /// <summary>
-    ///    <para>
-    ///       The IL instruction opcodes supported by the runtime.
```

Comment:
Nit: This variant of this comment formatting looks slightly better.

Reply:
Am I allowed to force push into this branch, provided I don't touch the first commit? (If I'm to fix this nit, I don't think it warrants a new commit.)

Reply:
yes, it is fine to force push